packages/ui/packages/Modal/test/index.spec.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+
+import { Wrapper } from '../../../test/test.helpers';
+import Modal from '../index';
+
+
+describe('Modal ui component tests', () => {
+  it('Should render with default size and center position at default', () => {
+    const { getByTestId } = render(
+      <Wrapper><Modal visibility /></Wrapper>,
+    );
+
+    expect(getByTestId('modal')).toHaveStyle({
+      width: 'calc(70 * 1rem)',
+      top: '50%',
+      transform: 'translateY(-50%)',
+    });
+  });
+
+  // it('Should change the class when  get different types of position props', () => {
```

Comment:
fix the title : `// it('Should change style when  get different types of position props', () => {`